src/main/java/org/opensearch/plugin/insights/rules/model/healthStats/QueryInsightsHealthStats.java

```diff
@@ -53,14 +62,16 @@ public QueryInsightsHealthStats(final StreamInput in) throws IOException {
     public QueryInsightsHealthStats(
         final ThreadPool.Info threadPoolInfo,
         final int queryRecordsQueueSize,
-        final Map<MetricType, TopQueriesHealthStats> topQueriesHealthStats
+        final Map<MetricType, TopQueriesHealthStats> topQueriesHealthStats,
+        final Map<String, Long> fieldTypeCacheStats
     ) {
         if (threadPoolInfo == null || topQueriesHealthStats == null) {
             throw new IllegalArgumentException("Parameters cannot be null");
         }
         this.threadPoolInfo = threadPoolInfo;
         this.queryRecordsQueueSize = queryRecordsQueueSize;
         this.topQueriesHealthStats = topQueriesHealthStats;
+        this.fieldTypeCacheStats = fieldTypeCacheStats != null ? fieldTypeCacheStats : ZEROS_FIELD_TYPE_CACHE_STATS;
```

Comment:
Why can't this be empty map?

Reply:
I am concerned about streaming data between nodes. We are using read map in & out so should be able to handle missing keys.

Default value in `toXContent()` ensures output always contains desired keys. 
```
        // Write field type cache stats
        builder.startObject(FIELD_TYPE_CACHE_STATS);
        for (String key : List.of("hits", "misses", "bytes")) {
            builder.field(key, fieldTypeCacheStats.getOrDefault(key, 0L));
        }
```

I will do some testing to verify

Reply:
Changed to empty map